### PR TITLE
changes to fetch the config report data

### PR DESCRIPTION
**Pull request by amolpati30**
Written some entities and views to schedule the Ansible job and fetch data from the configuration report log.

Dependent PR: https://github.com/SatelliteQE/robottelo/pull/15770

### CODE REVIEW

**Comment by amolpati30** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_ansible.py -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors
robottelo: 15770

File: airgun/entities/host_new.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import time
 
 from navmazing import NavigateToSibling
+from selenium.webdriver.common.by import By
```
Comment (omkarkhatavkar) [inline airgun/entities/host_new.py:4]: No direct selenium imports

**Review by omkarkhatavkar — CHANGES_REQUESTED** [review]

**Comment by amolpati30** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_ansible.py::TestAnsibleCfgMgmt -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors
robottelo: 15770

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 345
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/ui/test_ansible.py::TestAnsibleCfgMgmt -k test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors --external-logging
Test Result : ========== 1 passed, 17 deselected, 38 warnings in 2764.55s (0:46:04) ==========
```

File: airgun/entities/host_new.py (hunk 1 above)
Comment (amolpati30) [reply airgun/entities/host_new.py]: Removed this part

File: airgun/entities/activationkey.py [hunk 2]
```diff
@@ -86,6 +86,13 @@ def remove_host_collection(self, entity_name, hc_name):
         view.flash.assert_no_error()
         view.flash.dismiss()
 
+    def enable_repository(self, entity_name, repo_name):
+        view = self.navigate_to(self, 'Edit', entity_name=entity_name)
+        view.repository_sets.click()
+        view.repository_sets.search(repo_name)
+        view.repository_sets.check_box.click()
+        view.repository_sets.actions.fill('Override to Enabled')
```
Comment (omkarkhatavkar) [inline airgun/entities/activationkey.py:94]: why fill not working for you ? Can you please override that if that is not working for you ?

**Review by omkarkhatavkar — APPROVED** [review]
ACK, Overall looks good. One minor change

**Review by Gauravtalreja1 — APPROVED** [review]

**Review by omkarkhatavkar — APPROVED** [review]

**Pull request merged**